Core/tooling/fix-bundles.js

```diff
@@ -70,65 +69,48 @@ const replace = (libSrc, regex, replacement) => {
   return libSrc.replace(regex, replacement);
 }
 
-let openpgpLib = fs.readFileSync('./node_modules/openpgp/dist/node/openpgp.js').toString();
-const openpgpLibNodeDev = openpgpLib; // dev node runs without any host, no modifications needed
-
-/*
-openpgpLib = replace( // rsa decrypt on host
-  openpgpLib,
-  /[a-z0-9A-Z_]+\.default\.rsa\.decrypt\(c, n, e, d, p, q, u\)/,
-  `await hostRsaDecryption(dereq_asn1, _bn2, data_params[0], n, e, d, p, q)`
+// update openpgp code to use some native functionality
+let entrypointBareSrc = fs.readFileSync(`${bundleRawDir}/entrypoint-bare.js`).toString();
+
+// entrypointBareSrc = replace( // rsa decrypt on host
+//   // todo: use randomPayload value on iOS side
+//   entrypointBareSrc,
+//   /publicKey\.rsa\.decrypt\(c, n, e, d, p, q, u, randomPayload\)/,
+//   `await hostRsaDecryption(global.dereq_asn1, bn, c, n, e, d, p, q)`
+// );
+/* disabled because it works faster without this change */
+// entrypointBareSrc = replace( // rsa verify on host
+//   entrypointBareSrc,
+//   /return publicKey\.rsa\.verify\(hashAlgo, data, s, n, e, hashed\)/, `
+//   // returns empty str if not supported: js fallback below
+//   const computed = await coreHost.modPow(s.toString(10), e.toString(10), n.toString(10));
+//   return computed
+//     ? new bn.default(computed, 10).toArrayLike(Uint8Array, 'be', n.byteLength())
+//     : await publicKey.rsa.verify(hashAlgo, data, s, n, e, hashed);`
+// );
```

Comment:
@tomholub I disabled these code changes for decrypting and verifying changes on host and `testRsaPerformance` test still passes (https://github.com/FlowCrypt/flowcrypt-ios/blob/feature/issue-1379-openpgpjs-5/FlowCryptAppTests/Core/FlowCryptCoreTests.swift#L391)

Probably `openpgp` had some performance improvements and we don't need to use iOS code for this functionality?

Reply:
The test may have fairly benevolent maximum time allowed to not be too flaky. Could you see what are the individual operation times on v4 vs v5 as it is now?

Reply:
Just to understand:
 - `rsa verify on host` was not replaced because it's faster when not replacing it - that sounds good. In that case, this chunk of code could be removed instead of just commented out. How much faster exactly?
 - `rsa decrypt on host` - could you also check and compare the time before & after? using rsa4096 key like in the performance test.